controllers/cloud.redhat.com/providers/deployment/impl.go

```diff
@@ -270,6 +271,8 @@ func initDeployment(app *crd.ClowdApp, env *crd.ClowdEnvironment, d *apps.Deploy
 
 	d.Spec.Template.Spec.InitContainers = ics
 
+	d.Spec.Template.Spec.TerminationGracePeriodSeconds = pod.TerminationGracePeriodSeconds
```

Comment:
I _think_ we need to set the default here to 30, because otherwise we will get 0 which would change the default behaviour on the pods. We should have a test that tests the non-setting option too to ensure the default is what we expect.

Reply:
I'll add a test and update if we need to!

Reply:
I pushed a couple more tests, one of which completely leaves off the `terminationGracePeriodSeconds` in the clowdapp, and the assertion checks that it is correctly set to 30.